terraform/variables.tf

```diff
@@ -12,24 +12,30 @@ variable "remote_state_bucket" {
 }
 
 variable "state_prefix" {}
+variable "workspace_key_prefix" {}
+variable "state_file_name" {}
 
-# These vpcs are configured for development. Preprod and prod are overridden in the vars file.
-variable "vpc_cidr" {
-  default = {
-    eu-west-2 = "10.75.0.0/16" #London
-  }
+# Required for access from existing Concourse instances
+variable "dev_management_cidrs" {
+  type    = list(string)
+  default = ["10.50.17.0/24","10.50.19.0/24","10.50.21.0/24"]
```

Comment:
We should be pulling this in remotely. Also be aware this is a public repo

Reply:
Removed the dev_management_cidrs hard-coded list and replaced with reference to the remote state file.